src/_helpers/theme.js

```diff
@@ -46,4 +46,16 @@ const getLineHeight = (currentTheme, textSize) => {
   }
 };
 
-export { makePxValue, getColorKeys, getColor, getVariantColorKeys, getLineHeight };
+const getThemeColors = () => {
+  const flatten = (obj) => {
```

Comment:
Flatten can't be implemented inside `theme.js`. Put inside `_helpers/flatten.js`.

Reply:
This is not exactly flattening, may be I named the function wrong. It just gets the deepest nested value. Not sure what name should it be given.

Reply:
Is this needed after changing prop to `fill: colors.sapphire[800]`?

Reply:
Yes, we will need this. `fill: colors.sapphire[800]` is the default value, where as fill can be any value from `getThemeColors` i.e `fill: PropTypes.oneOf(getThemeColors())`.